lib/registration.dart

```diff
@@ -326,16 +328,44 @@ class _RegistrationState extends State<Registration> {
                                     ),
                                   ),
                                   haveCar == true
-                                      ? Container(
-                                          width: width * 0.85,
-                                          child: CarPlateForm(
-                                            onChanged: (plate) {
-                                              carPlate = plate;
-                                            },
-                                          ),
-                                        )
+                                      ? ListView.separated(
+                                          separatorBuilder: (context, index) =>
+                                              Divider(
+                                                color: Colors.white,
+                                              ),
+                                          shrinkWrap: true,
+                                          itemCount: carPlates.length,
+                                          itemBuilder: (BuildContext context,
+                                              int index) {
+                                            return Dismissible(
+                                              key: Key(index.toString()),
+                                              child: Container(
+                                                  width: width * 0.85,
+                                                  child: CarPlateForm(
+                                                    key: ObjectKey(carPlates
```

Comment:
Why are we using a key for `CarPlateForm` ? also, if we have 2 car plates won't that duplicate the key since it is dependent on the length of `carPlates` ? 

If the key is necessary, maybe we can use a combination of the index and the car plate value itself so that it is always unique

Reply:
it depends on carPlates.length so the first takes key 1 the second takes key 2 and so on

Reply:
Regarding the render overflow it is because my old android device, this isn't the emulator. its model is[ Lenovo Vibe K5 Plus](https://www.gsmarena.com/lenovo_vibe_k5_plus-7947.php) which has 5.0 inches screen

Reply:
Regarding the key, using `carPlates.length` will return the same value for both widgets, as the length of `carPlates` is the same when both widgets are built. Should we use `index` as a key instead ? 
In any case, If it isn't causing any errors, we can leave it as it is for now.

For the render overflow, I will try to see if I can fix the car plates for small devices such as that one, in case any user has a similar one.